swagger.json

```diff
@@ -1118,6 +1118,59 @@
             }
           }
         },
+        "/fleet/drivers/{driver-id}/log_edits": {
+            "get": {
+              "tags": [
+                "Fleet"
+              ],
+              "summary": "/fleet/drivers/{driver-id:[0-9]+}/log_edits",
+              "description": "Get a summary of the carrier edits for a specified driver. Note: This does not include edits made by driver",
+              "operationId": "getLogEdits",
+              "parameters": [
+                {
+                    "$ref": "#/parameters/accessTokenParam"
+                },
+                {
+                    "name": "driver-id",
+                    "in": "path",
+                    "required": true,
+                    "description": "ID (integer) of the driver with HOS logs.",
+                    "type": "number",
+                    "format": "int64"
+                },
+                {
+                "name": "created_at_start_ms",
```

Comment:
created_at_start_ms and created_at_end_ms are query parameters and should be spinal-case.